script/lint

```diff
@@ -19,3 +19,4 @@ SOURCES="*.py octodns/*.py octodns/*/*.py tests/*.py"
 
 pycodestyle --ignore=E221,E241,E251,E722,W504 $SOURCES
 pyflakes $SOURCES
+flake8 . --count --exclude=./env/* --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Running into problems with the way this does `.` when I have extra virtualenvs in my working directory. It tries to run on all the code in them.

```suggestion
flake8 $SOURCES --count --select=E9,F63,F7,F82 --show-source --statistics
```

Ideally Would like to remove the `--select` eventually and get all the warnings fixed. Can do that in subsequent PR(s).